src/requirements/data/colleges/en.ts

```diff
@@ -152,6 +152,7 @@ const engineeringRequirements: readonly CollegeOrMajorRequirement[] = [
     source:
       'https://www.engineering.cornell.edu/students/undergraduate-students/curriculum/undergraduate-requirements',
     fulfilledBy: 'self-check',
+    allowCourseDoubleCounting: true,
```

Comment:
I don't think this can be double counted.

Reply:
removed!